app/src/main/java/com/example/sabonit/CartActivity.java

```diff
@@ -98,9 +100,23 @@ public void onProductDeleteClick(View view, int position) {
     }
 
     public void goToOrder(View view) {
```

Comment:
Very log function, needs to be split into some sub-functions (or use helper functions).
Take a look on the general comment on the entire request

Reply:
Yes I'll told Eilon we will work on that